respect-datalayer-db/src/commonMain/kotlin/world/respect/datalayer/db/school/daos/PersonEntityDao.kt

```diff
@@ -155,5 +155,23 @@ interface PersonEntityDao {
             """
     )
     suspend fun getAllUsers(sourcedId: String): List<PersonEntity>
+    @Transaction
```

Comment:
This query needs flipped: what it does now is looking at every person in the database, and seeing if they are related. What should happen: select PersonEntity.* WHERE PersonEntity.uidHash IN (
SELECT PersonRelatedPersonEntity.prpOtherPersonUidNum
FROM ..
WHERE PersonRelatedPersonEntity.prpPersonUidNum = :guidHash
)